### DISCUSSION TITLE
How to use Sidebar Search Plugin?

### DISCUSSION THREAD

**Discussion by TiedtTech** [ColorlibHQ/AdminLTE#3815] [Q&A] [answered]
In my project I'm using AdminLTE from https://cdnjs.cloudflare.com/ajax/libs/admin-lte/3.0.5/js/adminlte.min.js but it doesn't load SidebarSearch.js on it.

How do I use this plugin in my project? Does anyone have a simple example to show me, because in the documentation itself it is not clear how to use this plugin.

mainly at this point:

`$('[data-widget="sidebar-search"]').SidebarSearch('toggle')`

Where in HTML / JS should I put this line of code?

**Comment by dfsmania** [comment, marked as answer]
If you are referencing this [SidebarSearch](https://adminlte.io/docs/3.1/javascript/sidebar-search.html) plugin, I think it is only available for **AdminLTE v3.1.0** (and maybe upper versions). Note you are using version `3.0.5` from the CDN.

**Reply by TiedtTech** [reply]
I understood my fault.
Where in the code should I put the line below and how do I configure the plugin. I looked in the 3.1.0 sources and couldn't find the code below the documentation.

`$('[data-widget="sidebar-search"]').SidebarSearch('toggle')`

**Reply by dfsmania** [reply]
That line is `jQuery` code, you should put it inside a `script` tag at the end of the `body` tag of your page, example:

```html
<html ...>
  <head>
    ...
  </head>
  <body>
    ...
    <script>
      $(document).ready(function()
      {
        // Example to toggle dropdown list of the search bar...
        $('[data-widget="sidebar-search"]').SidebarSearch('toggle');

        // Example to initialize the plugin with options...
        let options = {
          arrowSign: '=>',
          minLenght: 4,
          ...
        };

        ("[data-widget="sidebar-search"]").SidebarSearch(options);
      });
    </script>
  </body>
</html>
```

I think they assume you know about `jQuery/Javascript` on the documentation.

**Reply by TiedtTech** [reply]
Tks
